src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -12,6 +12,7 @@ import { LoadingSpinner } from '../../components';
 import { colors, Icon, normalize } from '../../config';
 
 export const ARShowScreen = ({ navigation, route }) => {
+  const [startAnimation, setStartAnimation] = useState(false);
```

Comment:
i think it would be better to start the animation instantly, why should users want to not have something animated?

Reply:
I can do this, but I think the animation will work in conjunction with the sound. I think it will be a bit interesting if users start listening to the sound before they realise what is happening.